### PR TITLE
chore: Prepare common module for publishing

### PR DESCRIPTION
**Pull request by Nurkambay**
## Checklist:
- [x] Each of my commits contains one meaningful change
- [x] I have performed rebase of my branch on top of the development
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: server/common/pom.xml [hunk 1]
```diff
@@ -43,6 +43,22 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     </properties>
 
+    <name>Broadcom COBOL LS Common Module</name>
```
Comment (Nurkambay) [inline server/common/pom.xml:46]: @zeibura  @zimlu02 please check the name of the module and the description

File: server/common/pom.xml [hunk 2]
```diff
@@ -43,6 +43,22 @@
...
+    <description>Provides dialect's support for 3rd party companies</description>
+    <url>https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support</url>
+
+    <licenses>
```
Comment (Nurkambay) [inline server/common/pom.xml:50]: @zimlu02 please check if Eclipse Public License - v 2.0 fits our needs

File: server/common/pom.xml [hunk 3]
```diff
@@ -43,6 +43,22 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     </properties>
 
+    <name>Broadcom COBOL LS Common Module</name>
+    <description>Provides dialect's support for 3rd party companies</description>
+    <url>https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support</url>
```
Comment (Nurkambay) [inline server/common/pom.xml:48]: @zimlu02 please take a look on url tag. It should be an url to the project. Now it is COBOL LS Marketplace url

Comment (zeibura) [reply server/common/pom.xml:48]: Provides dialect support for third-party companies

**Review by zeibura — CHANGES_REQUESTED** [review]
see inline

File: server/common/pom.xml (hunk 2 above)
Comment (ishche) [reply server/common/pom.xml:50]: But we are EPL2 https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/development/LICENSE.md
Why do you want to publish it under apache?

**Comment by Nurkambay** [comment]
> see inline

Fixed

File: server/common/pom.xml (hunk 2 above)
Comment (Nurkambay) [reply server/common/pom.xml:50]: Fixed.

File: server/common/pom.xml [hunk 4]
```diff
@@ -43,6 +43,104 @@
...
+            <url>https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol/blob/development/LICENSE.md</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <url>https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol.git</url>
+        <connection>scm:git:https://github.com/eclipse-che4z/che-che4z-lsp-for-cobol.git</connection>
+        <developerConnection>scm:git:ssh://github.com/eclipse-che4z/che-che4z-lsp-for-cobol.git</developerConnection>
+        <tag>2.0.1</tag>
```
Comment (Nurkambay) [inline server/common/pom.xml:62]: or release-2.0.1 ?

File: server/common/pom.xml [hunk 5]
```diff
@@ -43,6 +43,104 @@
...
+    </scm>
+
+    <developers>
+        <developer>
+            <organization>Broadcom, Inc</organization>
```
Comment (Nurkambay) [inline server/common/pom.xml]: @zimlu02 please check company name and url

**Review by zimlu02 — APPROVED** [review]

**Review by zeibura — APPROVED** [review]

**Review by ishche — APPROVED** [review]

**Pull request merged**